domain/src/main/java/life/qbic/usermanagement/policies/PasswordEncryptionPolicy.java

```diff
@@ -25,7 +25,7 @@ public class PasswordEncryptionPolicy {
   private static final int HASH_INDEX =
       2; // the index of the hash content in the encoded password String
   private static final int ITERATIONS =
-      4242; // the iteration count used for the encryption algorithm
+      10000; // the iteration count used for the encryption algorithm
```

Comment:
Increases readability:
```suggestion
      10_000; // the iteration count used for the encryption algorithm
```